config/nrfconnect/overlay-sample-defaults.conf

```diff
@@ -0,0 +1,96 @@
+#
+#   Copyright (c) 2020 Nordic Semiconductor
```

Comment:
Should this file have the CHIP copyright?

Reply:
given where it lives, I think so...

Reply:
Ok, it's done.

Reply:
@BroderickCarlin could you verify if you are ok with this now and approve or request further changes?